### PR TITLE
frontusb: miscellaneous compile fixes

### PR DESCRIPTION
**Pull request by gmacd**

### CODE REVIEW

File: sys/src/cmd/frontusb/cam/cam.c [hunk 1]
```diff
@@ -232,7 +232,7 @@ fsread(Req *req)
 	}
 	f = req->fid->file;
 	c = f->aux;
-	if(req->fid->aux == nil || req->ifcall.offset == 0)
+	if(req->fid->aux == nil || req->ifcall.offset == 0) {
```
Comment (rminnich) [inline sys/src/cmd/frontusb/cam/cam.c:235]: wow, this is a nice fix by spatch. That existing code was an accident waiting to happen.

File: sys/src/cmd/frontusb/cam/ctl.c [hunk 2]
```diff
@@ -216,7 +216,7 @@ penumwrite(Cam *c, int term, Param *p, char **f, int nf)
 }
 
 char *
-pformatread(Cam *c, int, Param *)
+pformatread(Cam *c, int term, Param *param)
```
Comment (rminnich) [inline sys/src/cmd/frontusb/cam/ctl.c:219]: This is an unfortunate spatch thing we can't fix, but it's a shame we can't say, Rust style, _term, and _param to indicate we won't use these.

In other parts of the tree we do this to indicate unused args.

File: sys/src/cmd/frontusb/disk/scsireq.c [hunk 3]
```diff
@@ -273,14 +274,14 @@ SRread(ScsiReq *rp, void *buf, i32 nbytes)
...
-	    rp->sense[2] & Sd2ili && n > 0)
+	    (rp->sense[2] & Sd2ili && n > 0))
 		rp->data.count = nbytes - n;
 	else
 		return -1;
 	n = rp->data.count;
 	if (!rp->readblock++ || debug)
 		fprint(2, "SRread: tape data count %ld%s\n", n,
-			(rp->sense[2] & Sd2ili? " with ILI": ""));
+			(rp->sense[2] & Sd2ili? " with IQLockLI": ""));
```
Comment (rminnich) [inline sys/src/cmd/frontusb/disk/scsireq.c:284]: Is this right? ILI -> IQLockLI?

File: sys/src/cmd/frontusb/lib/dump.c [hunk 4]
```diff
@@ -12,10 +12,10 @@ static char* cnames[] =
 	"none", "audio", "comms", "hid", "",
 	"", "", "printer", "storage", "hub", "data"
 };
-static char* devstates[] =
+/*static char* devstates[] =
```
Comment (rminnich) [inline sys/src/cmd/frontusb/lib/dump.c:15]: why comment this out? usage warning?

File: sys/src/cmd/frontusb/lib/usb.h [hunk 5]
```diff
@@ -344,8 +344,8 @@ struct DEp
 #define dprint if(usbdebug)fprint
 #define ddprint if(usbdebug > 1)fprint
 
-#pragma	varargck	type  "U"	Dev*
-#pragma	varargck	argpos	devctl	2
+//#pragma	varargck	type  "U"	Dev*
```
Comment (rminnich) [inline sys/src/cmd/frontusb/lib/usb.h:347]: just remove these.

File: sys/src/cmd/frontusb/cam/cam.c (hunk 1 above)
Comment (gmacd) [reply sys/src/cmd/frontusb/cam/cam.c:235]: It wasn't spatch - it was found by gcc and fixed by hand.

File: sys/src/cmd/frontusb/cam/ctl.c (hunk 2 above)
Comment (gmacd) [reply sys/src/cmd/frontusb/cam/ctl.c:219]: This was another manual change - detected by gcc, fixed by hand.  As such, we can precede with underscore.

File: sys/src/cmd/frontusb/disk/scsireq.c (hunk 3 above)
Comment (gmacd) [reply sys/src/cmd/frontusb/disk/scsireq.c:284]: This is definitely not right, and I have no idea how it got here.

File: sys/src/cmd/frontusb/lib/dump.c (hunk 4 above)
Comment (gmacd) [reply sys/src/cmd/frontusb/lib/dump.c:15]: It's not used.  Can remove entirely tbh.

File: sys/src/cmd/frontusb/lib/usb.h (hunk 5 above)
Comment (gmacd) [reply sys/src/cmd/frontusb/lib/usb.h:347]: sure.  there are other cases where it's been commented out in harvey.  these should be cleaned up later.

File: sys/src/cmd/frontusb/audio/audio.c [hunk 6]
```diff
@@ -264,7 +264,7 @@ main(int argc, char *argv[])
 	createfile(fs.tree->root, "volume", user, 0666, nil);
 
 	snprint(buf, sizeof buf, "%d.audio", audiodev->id);
-	postsharesrv(&fs, nil, "usb", buf);
+	//postsharesrv(&fs, nil, "usb", buf);
```
Comment (rminnich) [inline sys/src/cmd/frontusb/audio/audio.c:267]: I am guessing we want to wait on this until devshr is in?

Comment (gmacd) [reply sys/src/cmd/frontusb/audio/audio.c:267]: yeah

File: sys/src/cmd/frontusb/cam/video.c [hunk 7]
```diff
@@ -364,7 +366,8 @@ videoread(Req *r, Cam *c, int lock)
 void
 videoflush(Req *r, Cam *c)
 {
-	Req **rp;
+	// TODO requires 9pqueue
```
Comment (rminnich) [inline sys/src/cmd/frontusb/cam/video.c:369]: let's get 9pqueue now right?

File: sys/src/cmd/frontusb/cam/video.c [hunk 8]
```diff
@@ -53,14 +53,15 @@ pushframe(Cam *c, VFrame *v)
 	*l = v;
 	while(c->delreq != nil && c->actl != nil){
 		r = c->delreq;
-		c->delreq = (Req*)r->qu.next;
+		// TODO requires 9pqueue
+		//c->delreq = (Req*)r->qu.next;
```
Comment (rminnich) [inline sys/src/cmd/frontusb/cam/video.c:57]: it is time to uncomment this?

**Pull request closed without merging**